independent-publisher-connectors/Shopify/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1804 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "1.0.0",
+        "title": "Shopify",

```

Comment:
Hello @raybennett-msft,
I hope you are doing well. It turns out that we have a submission for Shopify and it is under development. As you know we can't have the same name for both connectors. May I ask you please to change the name? maybe to "Shopify IP"

Reply:
Hi @Amjed-Ayoub I do not see any open PR for Shopify, can you please point me to it? More out of curiosity if we can somehow combine our effort.

Reply:
@raybennett-msft It is an internal connector, which our engineers are working on and development the connector with Shopify themselves. this connector is in our pipeline for more than 2 years now and it is not open sourced.
Thank you so much for understanding.